src/AzureCacheForRedis.cs

```diff
@@ -41,7 +41,7 @@ public static async Task<ConfigurationOptions> ConfigureForAzureWithSystemAssign
     /// Throws on failure by default (configurable in the <see cref="ConfigureForAzureAsync"/> method).
     /// </summary>
     /// <param name="configurationOptions">The configuration to update.</param>
-    /// <param name="clientId">Client ID of the user-assigned managed identity.</param>
+    /// <param name="clientId">Client ID or resource ID of the user-assigned managed identity.</param>
     /// <param name="principalId">Principal (object) ID of the user-assigned managed identity.</param>
```

Comment:
MSAL does not support principalId for Managed Identity. Is this something you want us to support?

Reply:
@samsaha-ms can you comment on this? We pass the MI's object/principal ID as the username for the Redis connection, so I believe we use it in some capacity while validating tokens on the server side

Reply:
Yes, this is something related to what our extension does to interact with Redis, not related to MSAL.
It does two things 1. Acquire token from MSAL 2. Send AUTH request to Redis with principal-id and token to comply with redis command format "auth <username> <password>"and we validate this pair in server side.
We could actually drop this field from here, we could parse the acquired token and get the oid field in token and use that value while sending to redis.